models/general/entertainment/en/images.txt

```diff
@@ -1,4 +1,5 @@
 images of *|pictures of *|get me images on *|How does  * looks like|show me a picture of *|a picture of *|an image of *|show me a *|can you show me a *|visuals on *|visuals of *|image of *|picture of *|how would * look like|get me a glipmse of *|pictorial representation of *|fetch me some *|diagram of *
+!image: https://raw.githubusercontent.com/fossasia/susi_skill_data/master/models/general/entertainment/en/images.jpg
```

Comment:
Why is this image named "images.jpg"? Images should have the same name as the skill.

Just like many other images, the image here is not relevant for the skill.

All images need to be in a square format. So that they fit in easily into the cms. Please do compare your work with how images are used in Alexa (https://www.amazon.com/b?node=13727921011)